filecoin-proofs/tests/api.rs

```diff
@@ -479,6 +481,38 @@ fn create_seal<R: Rng, Tree: 'static + MerkleTreeTrait>(
         &phase1_output,
     )?;
 
+    Ok((piece_infos, phase1_output))
+}
+
+fn create_seal<R: Rng, Tree: 'static + MerkleTreeTrait>(
+    rng: &mut R,
+    sector_size: u64,
+    prover_id: ProverId,
+    skip_proof: bool,
+) -> Result<(SectorId, NamedTempFile, Commitment, tempfile::TempDir)> {
+    init_logger();
+
+    let (mut piece_file, piece_bytes) = generate_piece_file(sector_size)?;
+    let sealed_sector_file = NamedTempFile::new()?;
+    let mut unseal_file = NamedTempFile::new()?;
+    let cache_dir = tempfile::tempdir().expect("failed to create temp dir");
+
+    let arbitrary_porep_id = rng.gen();
```

Comment:
If you randomly generate this, then you won't be able to reuse the parents cache. So the price of regenerating the cache will be paid on every run, and more and more disk will be consumed. I think this should be a hardcoded constant for that reason.